### ISSUE TITLE
Norwegian is shown as online but it is not

### ISSUE THREAD

**Issue by ice1e0** [bcc-code/bcc-media-tools#29]
I created a new title and it shows that Norwegian is uploaded, but it is not.

![image](https://github.com/user-attachments/assets/f5b161e0-83bd-4817-8a4a-e769ae9eeda3)

![image](https://github.com/user-attachments/assets/889605cb-e2dc-4a92-a683-383baaade640)

**Comment by kkuepper** [comment]
Right. The original language is currently always shown as uploaded even though it isn't. The reason is that it caused problems (some of the unit tests fail), to have an empty list.
Not sure what's the best way to fix it.

**Comment by ice1e0** [comment]
@kkuepper maybe Show the language flag grey without color?

**Comment by ice1e0** [comment]
Just want to mention that this is still the case. Example:
No file is uploaded for the 22nd:
![image](https://github.com/user-attachments/assets/e9fef4cb-c426-475f-8cef-93d47d9c5f5c)
But it still shows the Norwegian flag for the 22nd
![image](https://github.com/user-attachments/assets/ce3000fb-bfa2-4250-9d59-72089f0616b0)
As soon as a language has been uploaded e.g. on the 23rd English, the Norwegian flag is shown grey as it should be.
